docs/troubleshooting/connection-string-missing.md

```diff
@@ -0,0 +1,35 @@
+---
+title: The connection string is missing
+description: Learn how to troubleshoot the error "ConnectionString is missing" during execution of your app.
+ms.date: 07/03/2024
+---
+
+# The specified name is already in use
+
+In an Aspire app, code identifies resources with an arbitrary string, such as "database". Code that is consuming the resource elsewhere must use the same string or it will fail.
```

Comment:
```suggestion
In .NET Aspire, code identifies resources with an arbitrary string, such as "database". Code that is consuming the resource elsewhere must use the same string or it will fail to correctly configure their relationships.
```